src/getting_started/linux_x86.md

```diff
@@ -75,6 +75,41 @@ MEYCIQCrc9cys35NeXwNAr8lYu8WPu0RiutkoAIWn+jYfYofPwIhAKPlNNsW//ykW8nX11KABNpWWYsN
 
 Execute the tool without any argument to check all the possible commands!
 
+## Running the `parsec-cli-tests.sh` script
+It's a script running simple end-two-end Parsec tests using parsec-tool and openssl
+
+As it uses the `parsec-tool`, `PARSEC_SERVICE_ENDPOINT` environment variable need to be set first.
+```
+export PARSEC_SERVICE_ENDPOINT=unix:$(pwd)/parsec.sock
+```
+
+Also `PARSEC_TOOL` environment variable should point to the parsec tool.
+```
+export PARSEC_TOOL=unix:$(pwd)/parsec-tool
+```
+or the path of the `parsec-tool` is included in the `PATH` environment variable.
```

Comment:
I would rephrase this paragraph. Something like:
If `parsec-tool` is not installed into a directory included in `PATH` then you also need to define `PARSEC_TOOL` environment variable with a full path to it:

export PARSEC_TOOL="\<full path\>/parsec-tool"

Reply:
Do you think use <full path> here or we can use `$(pwd)`?
as it was used in the previous section [here](https://parallaxsecond.github.io/parsec-book/getting_started/linux_x86.html#using-the-parsec-tool)
just for consistency and I think it's easier for the user to copy and paste.
@anta5010

Reply:
Yes, I think we can use $(pwd) in the example as you copy parsec-tool in the same directory in your quickstart script.